### PR TITLE
Repo badge

### PR DESCRIPTION
**Pull request by johnhiggs**
## Proposed changes

* Added badge info with example for GitHub markup and HTML to bottom of readme

## Types of changes

[//]: # (What types of changes does your code introduce to WebdriverIO?)
[//]: # (_Put an `x` in the boxes that apply_)

- [ ] Bugfix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [X] Documentation update

## Checklist

[//]: # (_Put an `x` in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your code._)

- [X] I have read the [CONTRIBUTING](https://github.com/webdriverio/webdriverio/blob/master/CONTRIBUTING.md) doc
- [ ] I have added tests that prove my fix is effective or that my feature works
- [X] I have added necessary documentation (if appropriate)
- [ ] I have added proper type definitions for new commands (if appropriate)

## Further comments

[//]: # (If this is a relatively large or complex change, kick off the discussion by explaining why you chose the solution you did and what alternatives you considered, etc...)

### CODE REVIEW

**Comment by johnhiggs** [comment]
@christian-bromann I moved it to bottom and fixed badge color to match wdio

**Comment by mgrybyk** [comment]
The image is not showing up, maybe because of some typo. Can you make sure it works locally in markdown for you?

**Comment by johnhiggs** [comment]
@mgrybyk I added sample for HTML snippet vs github wiki syntax

File: README.md [hunk 1]
```diff
@@ -109,3 +103,24 @@ This project is maintained by [awesome people](/AUTHORS.md) following a common [
 [MIT](/LICENSE-MIT)
 
 [![FOSSA Status](https://app.fossa.com/api/projects/git%2Bgithub.com%2Fwebdriverio%2Fwebdriverio.svg?type=large)](https://app.fossa.com/projects/git%2Bgithub.com%2Fwebdriverio%2Fwebdriverio?ref=badge_large)
+
+## Contributing
```
Comment (christian-bromann) [inline README.md]: The contributing guide should stay at the top

**Pull request merged**